pkg/handler/hndlrCnfg.bzl

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
i'd just call this file `config.bzl`. no need to repeat `handler` (nor to smash all the letters).